jnosql-mapping/jnosql-mapping-semistructured/src/main/java/org/eclipse/jnosql/mapping/semistructured/query/CustomRepositoryHandler.java

```diff
@@ -295,9 +339,17 @@ private Class<?> getGenericTypeFromParameter(Parameter parameter) {
         throw new IllegalArgumentException("Cannot determine generic type from parameter");
     }
 
-    private static boolean isLong(Method method) {
+    private static boolean returnsLong(Method method) {
         return method.getReturnType().equals(long.class) || method.getReturnType().equals(Long.class);
     }
 
+    private static boolean returnsInt(Method method) {
+        return method.getReturnType().equals(int.class) || method.getReturnType().equals(Integer.class);
+    }
+
+    private static boolean returnsBoolean(Method method) {
```

Comment:
The PDM says that you are not using this method, maybe you forget to do the backport on this point.

Reply:
Oh, yes. I removed support for boolean return as the Jakarta Data doesn't mandate it, it was only necessary to implement it in 1.1.x branch to pass the Data 1.0 TCK. Then Data 1.1 TCK change the test to return int instead of boolean to align with the spec requirements, so I removed support for boolean too. Should I keep support for boolean in the 1.2.x branch as in 1.1.x branch, even if the Data spec doesn't specify it?

Reply:
@otaviojava, what do you prefer? Should I keep the support for boolean return value in update/delete methods also in 1.2.x? 

In 1.1.x it was required to pass the Data 1.0 TCK for JPA but the spec document didn't require it and it won't be required in Data 1.1 TCK. 

Or it's better to drop the support in 1.2.x as I intended and strictly support only what is required by the Jakarta Data specification?

Reply:
I would go to drop it.